cognite/client/data_classes/_base.py

```diff
@@ -512,6 +512,11 @@ def dump(self, camel_case: bool = False) -> dict[str, Any]:
 T_CogniteFilter = TypeVar("T_CogniteFilter", bound=CogniteFilter)
 
 
+class NoCaseConversionPropertyList(list):
+    def as_reference(self) -> list[str]:
+        return list(self)
```

Comment:
Ugly fix but it serves a purpose: leave all type annotations as is.

Reply:
I actually like it. Very explicit.